presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -299,19 +301,19 @@ protected void assertAccessDenied(
             @Language("RegExp") String exceptionsMessageRegExp,
             TestingPrivilege... deniedPrivileges)
     {
-        executeExclusively(() -> {
-            try {
-                queryRunner.getAccessControl().deny(deniedPrivileges);
-                queryRunner.execute(session, sql);
-                fail("Expected " + AccessDeniedException.class.getSimpleName());
-            }
-            catch (RuntimeException e) {
-                assertExceptionMessage(sql, e, ".*Access Denied: " + exceptionsMessageRegExp);
-            }
-            finally {
-                queryRunner.getAccessControl().reset();
-            }
-        });
+        assertException(session, sql, ".*Access Denied: " + exceptionsMessageRegExp, deniedPrivileges);
+    }
+
+    protected void assertWrappedAccessDenied(@Language("SQL") String sql, @Language("RegExp") String exceptionsMessageRegExp, TestingPrivilege... deniedPrivileges)
+    {
+        assertException(getSession(), sql, exceptionsMessageRegExp, deniedPrivileges);
+    }
+
+    private void assertException(Session session, @Language("SQL") String sql, @Language("RegExp") String exceptionsMessageRegExp, TestingPrivilege[] deniedPrivileges)
+    {
+        assertThatThrownBy(() -> executeExclusively(session, sql, deniedPrivileges))
```

Comment:
This refactor is nice, but it has nothing to do with the commit. Please extract it to separate commit.

Reply:
done